src/contracts/interfaces/GPv2Authentication.sol

```diff
@@ -0,0 +1,11 @@
+// SPDX-License-Identifier: LGPL-3.0-or-newer
+pragma solidity ^0.6.12;
+
+/// @title Gnosis Protocol v2 Authentication interface
+/// @author Gnosis Developers
+interface GPv2Authentication {
+    /// @dev determines whether the provided address is an authenticated solver.
+    /// @param prospectiveSolver the address of prospective solver.
+    /// @return true when prospectiveSolver is an authenticated solver, otherwise false.
+    function isSolver(address prospectiveSolver) external view returns (bool);
```

Comment:
Hmm, maybe the method name should be a little more verbose:
```suggestion
    function isAuthenticatedSolver(address prospectiveSolver) external view returns (bool);
```

Just a suggestion, feel free to leave it as is or hear what the others think.

Reply:
I don't really like the redundancy involved in calling `authenticator.isAuthenticatedSolver`.

Reply:
*wont fix*